webvtt.html

```diff
@@ -3697,6 +3703,11 @@
     track">text tracks</a> that use these rules for a given <a>media element</a>, or other playback
     mechanism, are rendered together, to avoid overlapping subtitles from multiple tracks.</p>
 
+    <p class="note">Note that in HTML, audio elements don't have a visual rendering area and
```

Comment:
Drop the leading "Note that". Also, it's not really true that audio elements don't have a visual rendering area, you can make them any size you like with CSS, it's just that we've decided not to use that area. Perhaps put this more matter-of-factly, like "No cues will be rendered for audio elements..." or similar?